stylelint.config.js

```diff
@@ -3,25 +3,40 @@ const path = require('path')
 
 module.exports = {
 	configBaseDir: path.resolve(__dirname, 'node_modules'),
-	plugins: ['stylelint-csstree-validator', 'stylelint-selector-no-empty'],
+	plugins: ['stylelint-csstree-validator'],
 	extends: [
-		'stylelint-config-recommended-scss',
+		'stylelint-config-standard',
```

Comment:
migration guideline to extend from standard https://stylelint.io/user-guide/configure/#extends